beacon-chain/p2p/discovery.go

```diff
@@ -122,6 +124,52 @@ func (s *Service) startDiscoveryV5(
 	return listener, nil
 }
 
+// filter the peer coming from our dht
```

Comment:
Can you add a bit to the GoDoc for this? What does it mean to filter a peer?

Reply:
Ok done, added in a comment on it